kustomize/go.mod

```diff
@@ -17,3 +17,5 @@ exclude (
 	github.com/russross/blackfriday v2.0.0+incompatible
 	sigs.k8s.io/kustomize/api v0.2.0
 )
+
+replace sigs.k8s.io/kustomize/api => ../api
```

Comment:
Let's remove the line "replace ...". 

The process is that, change the api, then release a version of it, say v0.2.1.
Then update this file to use the new api version
```
sigs.k8s.io/kustomize/api v0.2.1
```